.nuspec/Xamarin.Forms.nuspec

```diff
@@ -42,9 +42,49 @@
         <dependency id="Tizen.NET" version="4.0.0"/>
       </group>
     </dependencies>
+	
+	<references>	
```

Comment:
Mixing tabs and spaces with the rest of the file.

Reply:
Fixed.